src/jit/alloc.cpp

```diff
@@ -110,7 +110,7 @@ class ArenaAllocator::SinglePagePool
     }
 };
 
-ArenaAllocator::SinglePagePool ArenaAllocator::s_pagePool;
+ArenaAllocator::SinglePagePool ArenaAllocator::s_pagePool = {};
```

Comment:
What error was this generating?

Reply:
C4815, see above....

Reply:
Thanks, somehow I missed that. Looks like the internal (.NET Framework I assume?) build is using `/W4` (or at least selectively enabled some warnings). Perhaps the CoreCLR build should do the same?